apps/employee-monitoring/src/components/sidebar-items/monitoring/scheduling-sheet/field/AddFieldSsModal.tsx

```diff
@@ -0,0 +1,303 @@
+  const openSelectScheduleModal = () => setSelectScheduleModalIsopen(true);
+
+  // close select schedule modal
+  const closeSelectScheduleModal = () => {
+    setSelectScheduleModalIsopen(false);
+  };
+
+  // schedule sheet store
+  const {
+    schedule,
+    selectedScheduleId,
+    setSelectedScheduleId,
+    getScheduleById,
+    getScheduleByIdFail,
+    getScheduleByIdSuccess,
+    loadingSchedule,
+  } = useScheduleSheetStore((state) => ({
+    schedule: state.schedule,
+    selectedScheduleId: state.selectedScheduleId,
+    setSelectedScheduleId: state.setSelectedScheduleId,
+    getScheduleById: state.getScheduleById,
+    getScheduleByIdSuccess: state.getScheduleByIdSuccess,
+    getScheduleByIdFail: state.getScheduleByIdFail,
+    loadingSchedule: state.loading.loadingSchedule,
+  }));
+
+  // use SWR
+  const {
+    data: swrSchedule,
+    isLoading: swrScheduleIsLoading,
+    error: swrScheduleError,
+  } = useSWR(
+    !isEmpty(selectedScheduleId) ? `/schedules/${selectedScheduleId}` : null,
+    fetcherEMS,
+    {
+      shouldRetryOnError: false,
+      revalidateOnFocus: false,
+    }
+  );
+
+  // set loading to true
+  useEffect(() => {
+    getScheduleById();
+  }, [selectedScheduleId]);
+
+  // set the schedule
+  useEffect(() => {
+    // success
+    if (!isEmpty(swrSchedule)) getScheduleByIdSuccess(swrSchedule.data);
+
+    // fail
+    if (!isEmpty(swrScheduleError))
+      getScheduleByIdFail(swrScheduleError.message);
+  }, [swrSchedule, swrScheduleError]);
+
+  // this portion is only for testing... please delete after
+  useEffect(() => {
+    // setSelectedScheduleId('b0235625-0ffc-4fb5-86ea-43a41b1d6a28');
+  }, [modalState]);
+
+  return (
+    <>
+      <Modal
+        open={modalState}
+        setOpen={setModalState}
+        size="xl"
+        steady
+        fixedHeight
+      >
+        <Modal.Header>
+          <h1 className="text-2xl font-medium">Add Field Scheduling Sheet</h1>
+        </Modal.Header>
+        <Modal.Body>
+          <>
+            <Modal
+              open={selectGroupModalIsOpen}
+              setOpen={setSelectGroupModalIsOpen}
+            >
+              <Modal.Header>
+                <h1 className="text-2xl font-medium">Select one (1) group</h1>
+              </Modal.Header>
+              <Modal.Body>
+                <select>
+                  <option>test</option>
+                </select>
+              </Modal.Body>
+              <Modal.Footer>
+                <div className="justify-end w-full">
+                  <Button variant="info" onClick={closeSelectGroupModal}>
+                    Cancel
+                  </Button>
+                </div>
+              </Modal.Footer>
+            </Modal>
+
+            <SelectFieldSchedSsModal
+              modalState={selectScheduleModalIsOpen}
+              setModalState={setSelectScheduleModalIsopen}
+              closeModalAction={closeSelectScheduleModal}
+            />
+            <form id="AddFieldSsForm">
```

Comment:
addFieldSsForm

Reply:
Changed it to addFieldSsForm and removed the extra form